src/ra_type_traits.h

```diff
@@ -29,14 +29,23 @@ namespace detail {
 /// <remarks><c>char16_t</c> and <c>char32_t</c> are not considered valid by this type_trait.</remarks>
 template<typename CharT>
 struct _NODISCARD is_char :
-    std::bool_constant<(std::_Is_character<CharT>::value || std::is_same_v<CharT, wchar_t>)>
+    std::bool_constant<((std::is_same_v<CharT, char> || std::is_same_v<CharT, signed char>)
```

Comment:
What I had before was non-standard (despite being in namespace `std`). plus now we don't have to worry about `CharT `being `unsigned char`. This `type_trait `was used mostly for conversions..